ramalama/console.py

```diff
@@ -0,0 +1,53 @@
+"""
+MIT License
+
+(C) 2024-2025 ramalama developers
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+"""
+import os
+import sys
+import locale
+import logging
+
+def is_locale_utf8():
+    """Check if the system locale is UTF-8."""
+    lang = os.getenv("LC_CTYPE", "") or os.getenv("LANG", "")
+    return "UTF-8" in lang.upper() or "utf8" in lang.lower()
+
+def supports_emoji():
+    """Detect if the terminal supports emoji output."""
+    if not is_locale_utf8():
+        return False  # Block emoji if UTF-8 is not in locale
+
+    if sys.platform == "win32": # works 32, 64 bits machines
+        # Windows Terminal, ConEmu, VSCode, and WSL support emoji
+        if os.getenv("WT_SESSION") or os.getenv("ConEmuANSI") or os.getenv("TERM_PROGRAM") == "vscode":
+            return True
+        return False  # Legacy cmd.exe and PowerShell lack proper emoji support
+
+    return True  # Most Linux/macOS terminals support emoji
+
+# Allow users to override emoji support via an environment variable
+EMOJI = os.getenv("RAMALAMA_FORCE_EMOJI", "1") != "0" and supports_emoji()
```

Comment:
Are you sure this shouldn't be an or?

FORCE won't force here, because of the AND condition.

Also, surely we should default to 0 and not force by default.

Reply:
Basically we want

if (RAMALAMA_FORCE_EMOJI is set) or (supports_emoji())